src/shared/services/commandInterpreterHelper.js

```diff
@@ -178,6 +180,42 @@ const availableCommands = [{
       })
   }
 }, {
+  name: 'style',
+  match: (cmd) => /^style(\s|$)/.test(cmd),
+  exec: function (action, cmdchar, put, store) {
+    const match = action.cmd.match(/:style\s*(\S.*)$/)
+    let param = match && match[1] ? match[1] : ''
+
+    if (param === '') {
+      // Todo: show popup
```

Comment:
I just thought it was a bit overkill since there is a task in trello for adding the popup. But sure I can add it